### PR TITLE
Module 1: Best Practice GitHub flow

### PR DESCRIPTION
**Pull request by alvinlouis29**
## I have made this possible by using the best Git and GitHub flow practices

### CODE REVIEW

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## Hi @alvinlouis29 :raised_hands:
Good job so far installing the linters, configuring the .gitignore file, and making a pull request. You have also written a good readme file :+1:, however there are some issues you need to work on before you go to the next project.

### Highlights:writing_hand:

* You have a descriptive Readme :heavy_check_mark:
* Your second branch has a descriptive name:heavy_check_mark:
## Required Changes :recycle:

- [ ]   Use your PR description as an opportunity to set the reviewer up for a review they will want to do, a review they will know how to review. Kindly go ahead create a PR.
- [ ] Please refer to this [template ](https://github.com/microverseinc/linters-config/tree/master/html-css)to set up html-css linters, to make it look green and passing successful.

     * #### Please check the comments under the review too.
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by abruzy — CHANGES_REQUESTED** [review]
```markdown
Hi @alvinlouis29, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._
_1. Kindly add a short summary to your PR explaining the changes introduced._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: .hintrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport",
+    "no-inline-styles:error"
+  ]
+}
```
Comment (carlos-ssh) [inline .hintrc:18]: - Kindly enable your linters into your pull request. After this request another code review from your dashboard.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @alvinlouis29, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
-Kindly check the changes requested below 👇🏼

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .hintrc (hunk 1 above)
Comment (alvinlouis29) [reply .hintrc:18]: Thank you Carlos

**Comment by Spayco** [comment]
Hello @alvinlouis29,

This is an additional review as a result of the second opinion request.

I can see there were some inconsistencies with reviews which may have resulted in some confusion, sorry about that,

The second reviewer only asked for you to change the description, and didn't include linters, so the third review should've approved your project since you've done that change, and make the linter change optional.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard @fernandorpm and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Comment by nidalaa** [comment]
Hi @alvinlouis29 as you exceeded your review requests limits, I made sure that we changed your activity status automatically. It is approved but you still need to go to your dashboard and submit it as complete.

Cheers!

**Pull request merged**